launcher/launcher/main.go

```diff
@@ -35,13 +35,22 @@ var launchSpec spec.LaunchSpec
 
 func main() {
 	var exitCode int
-	defer func() {
-		os.Exit(exitCode)
-	}()
 
 	logger = log.Default()
+	// default logger output to stderr
+	log.SetOutput(os.Stdout)
 	logger.Println("TEE container launcher initiating")
 
+	defer func() {
```

Comment:
These defers seem convoluted. Why not just put the os.Exit at the bottom of the code? We don't need the `recover` if we do that.

Reply:
This way we can output panic to cloud logging for better debug.